### PR TITLE
GO1P-20652: Index event_attendance user and event

### PR DESCRIPTION
**Pull request by luannguyengo1**

### CODE REVIEW

File: es/Schema.php [hunk 1]
```diff
@@ -973,13 +973,19 @@ class Schema
                 AttendanceStatuses::PENDING      => ['type' => self::T_INT],
             ],
         ],
+        'account'             => [
+            'properties' => self::ACCOUNT_MAPPING['properties'],
```
Comment (andytruong) [inline es/Schema.php]: should exclude `account.metadata`?

Comment (andytruong) [reply es/Schema.php]: or we may define ACCOUNT_LITE, which only need first name, last name, mail.

File: es/Schema.php [hunk 2]
```diff
@@ -951,6 +951,19 @@ class Schema
             ],
     ];
 
+    const USER_ENROLLED_PROPERTIES = [
```
Comment (mrsinguyen) [inline es/Schema.php]: USER_ENROLLED_PROPERTIES => ACCOUNT_LITE_PROPERTIES

File: es/Schema.php [hunk 3]
```diff
@@ -973,13 +986,19 @@ class Schema
                 AttendanceStatuses::PENDING      => ['type' => self::T_INT],
             ],
         ],
+        'user'             => [
```
Comment (mrsinguyen) [inline es/Schema.php]: user => account

**Pull request merged**